homeassistant/components/binary_sensor/mysensors.py

```diff
@@ -101,8 +101,16 @@ def name(self):
     @property
     def device_state_attributes(self):
         """Return device specific state attributes."""
+        try:
```

Comment:
Instead of these `try … except` statements, use `hasattr(self.gateway, 'server_address')` or `getattr(self.gateway, 'server_address', None)`